Content.Client/_DV/Curation/UI/Cwoink/CwoinkControl.xaml.cs

```diff
@@ -0,0 +1,250 @@
+using System.Linq;
+using System.Text;
+using Content.Client._DV.UserInterfaces.Systems.Cwoink;
+using Content.Client.Administration.Managers;
+using Content.Client.Administration.UI.CustomControls;
+using Content.Client.UserInterface.Controls;
+using Content.Shared.Administration;
+using Content.Shared.CCVar;
+using Robust.Client.AutoGenerated;
+using Robust.Client.Console;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Configuration;
+using Robust.Shared.Network;
+using Robust.Shared.Utility;
+
+namespace Content.Client._DV.Curation.UI.Cwoink;
```

Comment:
Using directive is unnecessary.
```suggestion
using System.Linq;
using System.Text;
using Content.Client._DV.UserInterfaces.Systems.Cwoink;
using Content.Client.Administration.Managers;
using Content.Client.Administration.UI.CustomControls;
using Content.Shared.Administration;
using Content.Shared.CCVar;
using Robust.Client.AutoGenerated;
using Robust.Client.Console;
using Robust.Client.UserInterface;
using Robust.Client.UserInterface.XAML;
using Robust.Shared.Configuration;
using Robust.Shared.Network;
using Robust.Shared.Utility;

namespace Content.Client._DV.Curation.UI.Cwoink;
```